### ISSUE TITLE
can't emerge dde-meta / packages need updating

### ISSUE THREAD

**Issue by Renquest00** [zhtengw/deepin-overlay#136]
TL/DR: Request for all packages to get a version upgrade as some are not building in a current gentoo environment.

So I started writing a log of my adventure trying to get Deepin installed on my computer. I thought it would be short enough for a issue notice, but i was wrong. I got all but 20 packages installed. 10 have not tried to install yet because I could not get dde-daemon to compile even after making an ebuild for the new version. The other packages I skipped did not seem as important as this one.

So this is a request for all the packages to get updated to more recent versions. That solved some of my issues but likely some of the packages I did not make new version ebuilds for are preventing later things from installing. 

Some important points that also need to be looked at:

qtxcb-private-headers package requires EAPI 8. It was required by the inherited qt5-build.eclass

several packages that were updated to EAPI 7 were not fully converted.
EAPI 7 removed the inherited versionator.eclass, instead built in commands are used.
replace_version_separator needs to be converted to ver_rs (not sure if it's a 1 to 1 replacement)

gnome-base/libgnome-keyring seems to have been replaced by app-crypt/libsecret

Annoyances I found:
some package (I think it's deepin-movie-reborn) required mvp to have dvd use flag. I don't have an optical drive and did not like having to enable that.
onboard requires python 3.8. This might have been fixed with an ebuild for a new version but I would have to investigate more than what I was doing which was copying the older ebuild with a version number for the most recent release. My system is currently built with only python 3.9 and python 3.10 installed so I'd prefer not to add the older python version to my system.
